src/lwc/rd2EditPaymentInformationModal/rd2EditPaymentInformationModal.js

```diff
@@ -0,0 +1,317 @@
+import { LightningElement, api, track } from 'lwc';
```

Comment:
This file has a lot of lines with an extra space in front of the normal indentation.  Could you please normalize the indents here?